pythainlp/tokenize/core.py

```diff
@@ -302,6 +303,8 @@ def subword_tokenize(
         * *tcc* (default) -  Thai Character Cluster (Theeramunkong et al. 2000)
         * *etcc* - Enhanced Thai Character Cluster (Inrut et al. 2001)
         * *wangchanberta* - SentencePiece from wangchanberta model.
+        * *dict* (default) - newmm word tokenizer with a syllable dictionary
+        * *ssg* - CRF syllable segmenter for Thai
```

Comment:
"dict" is not a default subword tokenization engine.

Current default is "tcc",
according to DEFAULT_SUBWORD_TOKENIZE_ENGINE constant in
https://github.com/PyThaiNLP/pythainlp/blob/dev/pythainlp/tokenize/__init__.py

"*dict* (default) - newmm..." should be just "* *dict* - newmm..."